config/rootfs/debos/rootfs.yaml

```diff
@@ -59,7 +60,11 @@ actions:
   - action: run
     description: add firmware files
     chroot: false
-    script: scripts/install-firmware.sh {{ $extra_firmware }}
+    {{ if $linux_fw_version }}
+    script: scripts/install-firmware.sh -v {{ $linux_fw_version }} -f {{ $extra_firmware }}
+    {{ else}}
+    script: scripts/install-firmware.sh -f {{ $extra_firmware }}
```

Comment:
Maybe move `{{ if $linux_fw_version }}` to the command line itself to avoid repeating it? Something like:
```
script: scripts/install-firmware.sh -f {{ $extra_firmware }}{{ if $linux_fw_version }} -v {{ $linux_fw_version }} {{ end }}
```